### PR TITLE
PMM-6234 Fix tests for feature build.

### PR DESCRIPTION
**Pull request by askomorokhov**
Related PRs:
 - https://github.com/percona/pmm-managed/pull/426

### CODE REVIEW

**Comment by AlekSi** [comment]
I will revert that line and merge it

File: services/supervisord/devcontainer_test.go [hunk 1]
```diff
@@ -221,10 +224,11 @@ func TestDevContainer(t *testing.T) {
 	})
 }
 
-func normalizeFullversion(info *version.PackageInfo) string {
+func normalizeFullversion(info *version.PackageInfo) (version string, isFeatureBranch bool) {
 	fullVersion := info.FullVersion
-	if os.Getenv("FEATURE_BRANCH") != "" {
+	isFeatureBranch = os.Getenv("FEATURE_BRANCH") != "" || strings.HasPrefix(fullVersion, "1:")
```
Comment (AlekSi) [inline services/supervisord/devcontainer_test.go:229]: We don't set `FEATURE_BRANCH` anywhere now, I will remove it

Comment (askomorokhov) [reply services/supervisord/devcontainer_test.go:229]: Cool.

File: .travis.yml [hunk 2]
```diff
@@ -41,13 +34,17 @@ env:
...
+    # There is no case when pmm-update is older than pmm-managed (pmm-update self-updates first),
+    # so there is no need to test with old PMM Server versions.
+    # However, there may be a need to use a different devcontainer image once
+    # https://jira.percona.com/browse/PMM-5451 is implemented.
+    - PMM_SERVER_IMAGE=perconalab/pmm-server:dev-latest
 
-    # the latest HEAD
-    - GO_VERSION=tip
+    # To test with some feature build
+    # - PMM_SERVER_IMAGE=perconalab/pmm-server-fb:PR-123-4567890
```
Comment (askomorokhov) [inline .travis.yml:47]: While seeing this line is commented, just want to warn that I removed `- GO_VERSION=tip` with the intention to revert whole  `.travis.yml`

Comment (AlekSi) [reply .travis.yml:47]: That should be a reason for rejecting PR

**Comment by AlekSi** [comment]
Waiting for CI

**Pull request merged**